File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/FaktiskInntektsgrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.InntektTypeCti;

import java.io.Serializable;

public class FaktiskInntektsgrunnlag implements Serializable {

    

    /** Akkumulert inntekt hittil i år. */
    private int faktiskeInntekterHittilIAr;

    /** Angir hvilken type inntekt som er akkumulert. */
    private InntektTypeCti inntektType = new InntektTypeCti();

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/FaktiskInntektsgrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.enum.InntekttypeEnum
import java.io.Serializable

class FaktiskInntektsgrunnlag : Serializable {
    /** Akkumulert inntekt hittil i år.  */
    var faktiskeInntekterHittilIAr = 0

    /** Angir hvilken type inntekt som er akkumulert.  */
    var inntektTypeEnum = InntekttypeEnum.UKJENT
}